README.md

```diff
@@ -6,10 +6,68 @@
 
 ## Introduction
 
-Frequenz gRPC API for monitoring and control of microgrids.
+The Frequenz Microgrid API is an API for managing and monitoring components
+within a microgrid. It offers various functionalities related to microgrid
+management, including metadata retrieval, component listing, setting power
+levels, and controlling the state of components. The API is structured around
+the concept of a microgrid, which is a localized energy system composed of
+various interconnected components.
 
-This repository provides the core gRPC and protobuf spec together with
-support for generating dedicated API packages for various languages.
+
+## Objective
+
+The objective of this API is to provide a standardized interface for
+interacting with and controlling components within a microgrid. It aims to
+enable efficient management and monitoring of the microgrid's components,
+including batteries, inverters, chargers, meters, and more. It facilitates
```

Comment:
would be good to refer to them as component categories here to introduce the term "category".

Reply:
Done, but I moved this part to the `Introduction` section. As a result, the `Objectives` section is a lot smaller now.